### PR TITLE
Reconcile multiple service monitors with multiple scrape paths

### PR DESCRIPTION
**Pull request by nb-ohad**
For each pod behind the `noobaa-mgmt` service we will scrape 3 paths:
1. `/metrics/web_server` - nodejs `web_server` process report + nooba reports
2. `/metrics/bg_workers` - nodejs `bg_workers` process report
3. `/metrics/hosted_agents`- nodejs `hosted_agents` process report

For each pod behind the `s3` service we will scrape one paths (on port 7001):
1. `/` - nodejs `endpoint` report

Signed-off-by: nb-ohad <mitrani.ohad@gmail.com>

### CODE REVIEW

**Comment by nb-ohad** [comment]
This PR is marked as [WIP] although the code is complete. The mark is there because the change was not tested yet no an OCS cluster

File: deploy/internal/service-mgmt.yaml [hunk 1]
```diff
@@ -4,6 +4,7 @@ metadata:
   name: SYSNAME-mgmt
   labels:
     app: noobaa
+    noobaa-mgmt: "true"
```
Comment (dannyzaken) [inline deploy/internal/service-mgmt.yaml:7]: maybe use a different key for the label as we already use `noobaa-mgmt` for the pod labels. will be less confusing.  maybe `moobaa-mgmt-svc: "true"`

File: deploy/internal/service-s3.yaml [hunk 2]
```diff
@@ -4,6 +4,7 @@ metadata:
   name: s3
   labels:
     app: noobaa
+    noobaa-s3: "true"
```
Comment (dannyzaken) [inline deploy/internal/service-s3.yaml:7]: same. maybe change to `noobaa-s3-svc: "true"`

Comment (dannyzaken) [reply deploy/internal/service-s3.yaml:7]: if you change the labels don't forget to change the selectors using them

Comment (nb-ohad) [reply deploy/internal/service-s3.yaml]: done

File: deploy/internal/service-mgmt.yaml (hunk 1 above)
Comment (nb-ohad) [reply deploy/internal/service-mgmt.yaml]: done

**Review by dannyzaken — APPROVED** [review]

**Pull request merged**